plugins/newspaper/class.tx_dlf_newspaper.php

```diff
@@ -91,22 +93,112 @@ public function calendar($content, $conf) {
 
             $issues[] = array (
                 'uid' => $resArray['uid'],
-                'title' => $resArray['title'],
+                'title' => $resArray['title'] ? $resArray['title'] : (strftime('%x', strtotime($resArray['volume']))),
```

Comment:
I am not sure, but I think this has to be
```suggestion
                'title' => !empty($resArray['title']) ? $resArray['title'] : !empty($resArray['volume']) ? $resArray['volume'] : (strftime('%x', strtotime($resArray['volume_sorting']))),
```
because `$resArray['volume']` holds the information from `@LABEL`, which are not properly ISO-encoded (i.e. "Spielzeit 1902/1903"). Only `$resArray['volume_sorting']` can be formatted using `strftime()`.